src/tests/fixtures/navigation.html

```diff
@@ -6,22 +6,6 @@
     <title>Turbo</title>
     <script src="/dist/turbo.es2017-umd.js" data-turbo-track="reload"></script>
     <script src="/src/tests/fixtures/test.js"></script>
-    <script>
-      class CustomLinkElement extends HTMLElement {
-        constructor() {
-          super()
-          this.attachShadow({ mode: 'open' })
-        }
-        connectedCallback() {
-          this.shadowRoot.innerHTML = `
-            <a href="${this.getAttribute('link')}">
-              ${this.getAttribute('text')}
-            </a>
-          `
-        }
-      }
-      window.customElements.define('custom-link-element', CustomLinkElement)
-    </script>
```

Comment:
Moving this to `src/tests/fixtures/test.js` since now this is used by two different tests.